core/feed.py

```diff
@@ -20,9 +20,10 @@ def render_feed(feed_id: str, plugin: Plugin, options: GlobalOptions, base_url:
                 id=episode.item_id,
                 title=episode.title,
                 media=Media(generate_url(episode, plugin, options, base_url), episode.content_length, type=episode.content_type),
-                summary=episode.description,
+                summary=episode.description.replace("\n", "<br>") if episode.description and options.html_newlines else episode.description,
```

Comment:
Seen on invidious descriptions that newlines come as \n and they are lost on the way. Since this issue could happen with any plugin, we will fix it here on the global code and keep html_newlines to disable it in case an unexpected regression is produced.